content/_data/changelogs/weekly.yml

```diff
@@ -93,6 +93,10 @@
       pull: 2683
     - type: rfe
       message: >
+        Searching in the <code>Build History</code> widget takes into account user preferences (case sensitivity by default).
```

Comment:
Likely it came from me in changelogs for early 2.x versions on this year. Since I am not an HTML expert, I have just started using the tag I knew. And nobody stopped me till 2.50 :(